charts/chatwoot/templates/_helpers.tpl

```diff
@@ -200,3 +199,28 @@ Set redis URL
     redis://:$(REDIS_PASSWORD)@{{ .Values.redis.host }}:{{ .Values.redis.port }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "custom-metrics.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "custom-metrics.chart" -}}
```

Comment:
`custom-metrics`?

Reply:
Yeah, forgot to remove unused parts of configuration